doc/dev/source/installation.rst

```diff
@@ -19,6 +19,8 @@ You will need some system libraries:
 * *libffi-devel*
 * *openldap-devel* (if you wish to use LDAP authentication)
 
+You will also need the GNU Compiler Collection (commonly known by the package name gcc).
```

Comment:
Also because I believe OSX uses clang.

On Thu, 11 Aug 2016, 11:05 Adrian, <notifications@github.com> wrote:

> In doc/dev/source/installation.rst
> <https://github.com/indico/indico/pull/2553#discussion_r74388956>:
>
> > @@ -19,6 +19,8 @@ You will need some system libraries:
> >  * *libffi-devel*
> >  * *openldap-devel* (if you wish to use LDAP authentication)
> >
> > +You will also need the GNU Compiler Collection (commonly known by the package name gcc).
>
> What about something like this?
>
> Since some of the python dependencies contain compiled libraries you need
> a compiler such as gcc.
>
> I don't think anyone really uses the term "GNU Compiler Collection"
> instead of just "gcc".
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/indico/indico/pull/2553/files/ec823cfcf3327c89f990b6f5e5002ae69b7f0c60#r74388956>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAKi9bvKVnhT7MhX_gSD359b9tjqvD_ks5qeuXQgaJpZM4Jh64t>
> .
>